.readthedocs.yml

```diff
@@ -2,17 +2,14 @@ version: 2
 build:
   os: ubuntu-20.04
   tools:
-    python: mambaforge-4.10
+    python: mambaforge-latest
+  commands:
```

Comment:
@pavelzw This example seems to be structured a little differently:
https://github.com/prefix-dev/pixi/blob/main/examples/readthedocs-extend/docs/.readthedocs.yaml

Reply:
I added them here:
[1cfafb5](https://github.com/Quantco/datajudge/pull/230/commits/1cfafb5060805f8e006bd22404c95749cf071a73)